### PR TITLE
Use Special GA account to track 100% events on Auth Pages

### PR DESCRIPTION
**Pull request by bkoval**
@kvas-damian @hakubo @rafalkalinski 

This enforces usage of a special, unsampled GA account on auth pages along with the primary acount, sampling at 10%.

### CODE REVIEW

**Comment by kvas-damian** [comment]
@bkoval make sure special tracker works correctly on mercury

**Comment by kvas-damian** [comment]
Please fix Unit tests and LGTM

File: server/views/_layouts/auth.hbs [hunk 1]
```diff
@@ -61,6 +61,7 @@
 		<!-- endbuild -->
 		<script>
 			M.prop('tracking', {{{json trackingConfig}}});
+			M.prop('isGASpecialWiki', true);
```
Comment (rafalkalinski) [inline server/views/_layouts/auth.hbs]: is it used anywhere?

**Comment by rafalkalinski** [comment]
LGTM

File: server/views/_layouts/auth.hbs (hunk 1 above)
Comment (bkoval) [inline server/views/_layouts/auth.hbs]: Yes.

**Pull request merged**